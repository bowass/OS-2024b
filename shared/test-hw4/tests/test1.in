42769
61163
19791
44302
9346
3710
22639
63413
21708
39926
53367
49535
45438
65404
42479
50313
17553
16766
28859
15300
62724
50449
7786
62178
17881
48048
51027
11052
59838
13478
64147
2607
4121
56394
33884
9263
31474
32637
7483
42841
55130
13120
14404
48948
27765
20752
10357
34313
45661
22960
6894
27326
45801
39052
33649
62569
41244
12524
3300
30943
61142
1696
9110
46057
9116
13929
33054
27166
41680
36622
55096
46903
28108
3940
62956
8450
3924
58016
23272
55252
54880
60907
2000
33168
20822
29488
18541
29414
47990
42218
5324
3816
4535
55805
507
41378
12015
30643
533
36335
17731
4202
62
49581
60064
1850
11105
32943
9041
17298
56049
24768
54014
30875
26739
39456
18074
53349
33455
15853
6787
39639
37050
8789
17573
45940
53677
48740
41547
25791
58638
59213
35582
47319
51490
51246
7813
31637
40146
55051
31511
6871
38475
48768
1151
48552
19857
13291
19171
63120
37021
59139
54146
32135
12845
55657
2036
35030
20139
40865
24264
38656
58202
726
45856
14356
43220
5740
54735
31647
2537
3605
25660
5777
34708
45763
11509
52424
63520
49272
54153
54439
27949
19423
51785
15489
47636
38338
47059
19917
65441
3844
40724
24383
27178
54217
20514
12949
34521
46920
472
27647
38204
22993
30366
36134
50880
46120
33969
9414
62325
19229
14920
51196
61820
44765
47671
41094
53559
23793
16221
63979
21677
60744
1858
47532
4944
12918
35987
32478
61740
12272
46100
35055
62665
48604
9552
24770
19483
7102
13961
44326
58725
19951
50642
8794
21858
37531
58484
6437
44053
25360
8444
46509
45701
2140
45238
34737
35076
53149
12025
20907
37436
23285
14268
18430
52819
9365
33346
62705
39351
10389
59313
13892
35162
37132
2473
44561
51165
15838
44148
44315
40089
39125
58509
56116
20035
24749
50348
53143
38211
9093
24106
31134
32123
29968
13339
4362
184
59291
37713
49461
3450
12594
62308
35283
57957
37593
28456
4782
2101
42719
46881
59368
11245
57061
37252
23343
56032
51474
8888
22768
35505
20857
16746
57259
63278
40674
4010
27403
45576
17005
52621
38644
32869
33572
39304
24113
44856
13249
33733
12786
24234
26670
64792
11859
58281
26897
5900
6146
22445
22752
59295
43031
50880
15100
20202
40061
61616
30399
14935
26455
56896
33636
47147
38552
57676
2052
25769
2418
53593
13658
60795
21013
57971
19432
24339
205
44654
44801
16501
64965
24141
53878
15409
14377
52234
38561
24164
24918
10119
14400
27557
17704
36327
13037
38738
9743
53050
46871
7207
50939
11736
16581
19156
16622
64078
31990
48570
33282
18993
24493
23873
23682
33273
44298
23857
7966
35322
33850
5380
33088
46587
59413
64318
52462
36879
10311
25840
3808
26529
46611
48451
52901
2660
53150
27795
56085
27642
65463
29674
35092
61165
60381
24381
15497
28240
63968
51085
46685
28411
8859
48885
54527
61740
15613
19570
25976
11732
21739
35334
19955
14844
18082
35060
31986
7320
15501
64304
58953
44322
31275
65187
30582
50947
50486
13876
50423
37916
65292
63584
365
17461
45267
18051
42832
51613
64957
60294
47111
25648
25729
38780
32459
37672
16579
23146
63537
26950
12588