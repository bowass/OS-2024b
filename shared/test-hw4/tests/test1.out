Virtual address: 42769 Physical address: 17 Value: 0
Virtual address: 61163 Physical address: 491 Value: -70
Virtual address: 19791 Physical address: 591 Value: 83
Virtual address: 44302 Physical address: 782 Value: 43
Virtual address: 9346 Physical address: 1154 Value: 9
Virtual address: 3710 Physical address: 1406 Value: 3
Virtual address: 22639 Physical address: 1647 Value: 27
Virtual address: 63413 Physical address: 1973 Value: 0
Virtual address: 21708 Physical address: 2252 Value: 0
Virtual address: 39926 Physical address: 2550 Value: 38
Virtual address: 53367 Physical address: 2679 Value: 29
Virtual address: 49535 Physical address: 2943 Value: 95
Virtual address: 45438 Physical address: 3198 Value: 44
Virtual address: 65404 Physical address: 3452 Value: 0
Virtual address: 42479 Physical address: 3823 Value: 123
Virtual address: 50313 Physical address: 3977 Value: 0
Virtual address: 17553 Physical address: 4241 Value: 0
Virtual address: 16766 Physical address: 4478 Value: 16
Virtual address: 28859 Physical address: 4795 Value: 46
Virtual address: 15300 Physical address: 5060 Value: 0
Virtual address: 62724 Physical address: 5124 Value: 0
Virtual address: 50449 Physical address: 5393 Value: 0
Virtual address: 7786 Physical address: 5738 Value: 7
Virtual address: 62178 Physical address: 6114 Value: 60
Virtual address: 17881 Physical address: 6361 Value: 0
Virtual address: 48048 Physical address: 6576 Value: 0
Virtual address: 51027 Physical address: 6739 Value: -44
Virtual address: 11052 Physical address: 6956 Value: 0
Virtual address: 59838 Physical address: 7358 Value: 58
Virtual address: 13478 Physical address: 7590 Value: 13
Virtual address: 64147 Physical address: 7827 Value: -92
Virtual address: 2607 Physical address: 7983 Value: -117
Virtual address: 4121 Physical address: 8217 Value: 0
Virtual address: 56394 Physical address: 8522 Value: 55
Virtual address: 33884 Physical address: 8796 Value: 0
Virtual address: 9263 Physical address: 1071 Value: 11
Virtual address: 31474 Physical address: 9202 Value: 30
Virtual address: 32637 Physical address: 9341 Value: 0
Virtual address: 7483 Physical address: 9531 Value: 78
Virtual address: 42841 Physical address: 89 Value: 0
Virtual address: 55130 Physical address: 9818 Value: 53
Virtual address: 13120 Physical address: 10048 Value: 0
Virtual address: 14404 Physical address: 10308 Value: 0
Virtual address: 48948 Physical address: 10548 Value: 0
Virtual address: 27765 Physical address: 10869 Value: 0
Virtual address: 20752 Physical address: 11024 Value: 0
Virtual address: 10357 Physical address: 11381 Value: 0
Virtual address: 34313 Physical address: 11529 Value: 0
Virtual address: 45661 Physical address: 11869 Value: 0
Virtual address: 22960 Physical address: 12208 Value: 0
Virtual address: 6894 Physical address: 12526 Value: 6
Virtual address: 27326 Physical address: 12734 Value: 26
Virtual address: 45801 Physical address: 12009 Value: 0
Virtual address: 39052 Physical address: 12940 Value: 0
Virtual address: 33649 Physical address: 13169 Value: 0
Virtual address: 62569 Physical address: 13417 Value: 0
Virtual address: 41244 Physical address: 13596 Value: 0
Virtual address: 12524 Physical address: 14060 Value: 0
Virtual address: 3300 Physical address: 14308 Value: 0
Virtual address: 30943 Physical address: 14559 Value: 55
Virtual address: 61142 Physical address: 470 Value: 59
Virtual address: 1696 Physical address: 14752 Value: 0
Virtual address: 9110 Physical address: 14998 Value: 8
Virtual address: 46057 Physical address: 15337 Value: 0
Virtual address: 9116 Physical address: 15004 Value: 0
Virtual address: 13929 Physical address: 15465 Value: 0
Virtual address: 33054 Physical address: 15646 Value: 32
Virtual address: 27166 Physical address: 12574 Value: 26
Virtual address: 41680 Physical address: 16080 Value: 0
Virtual address: 36622 Physical address: 16142 Value: 35
Virtual address: 55096 Physical address: 9784 Value: 0
Virtual address: 46903 Physical address: 16439 Value: -51
Virtual address: 28108 Physical address: 16844 Value: 0
Virtual address: 3940 Physical address: 16996 Value: 0
Virtual address: 62956 Physical address: 5356 Value: 0
Virtual address: 8450 Physical address: 17154 Value: 8
Virtual address: 3924 Physical address: 16980 Value: 0
Virtual address: 58016 Physical address: 17568 Value: 0
Virtual address: 23272 Physical address: 17896 Value: 0
Virtual address: 55252 Physical address: 9940 Value: 0
Virtual address: 54880 Physical address: 18016 Value: 0
Virtual address: 60907 Physical address: 18411 Value: 122
Virtual address: 2000 Physical address: 18640 Value: 0
Virtual address: 33168 Physical address: 15760 Value: 0
Virtual address: 20822 Physical address: 11094 Value: 20
Virtual address: 29488 Physical address: 18736 Value: 0
Virtual address: 18541 Physical address: 19053 Value: 0
Virtual address: 29414 Physical address: 19430 Value: 28
Virtual address: 47990 Physical address: 6518 Value: 46
Virtual address: 42218 Physical address: 19690 Value: 41
Virtual address: 5324 Physical address: 19916 Value: 0
Virtual address: 3816 Physical address: 1512 Value: 0
Virtual address: 4535 Physical address: 20151 Value: 109
Virtual address: 55805 Physical address: 20477 Value: 0
Virtual address: 507 Physical address: 20731 Value: 126
Virtual address: 41378 Physical address: 13730 Value: 40
Virtual address: 12015 Physical address: 20975 Value: -69
Virtual address: 30643 Physical address: 21171 Value: -20
Virtual address: 533 Physical address: 21269 Value: 0
Virtual address: 36335 Physical address: 21743 Value: 123
Virtual address: 17731 Physical address: 6211 Value: 80
Virtual address: 4202 Physical address: 8298 Value: 4
Virtual address: 62 Physical address: 21822 Value: 0
Virtual address: 49581 Physical address: 2989 Value: 0
Virtual address: 60064 Physical address: 22176 Value: 0
Virtual address: 1850 Physical address: 18490 Value: 1
Virtual address: 11105 Physical address: 7009 Value: 0
Virtual address: 32943 Physical address: 22447 Value: 43
Virtual address: 9041 Physical address: 14929 Value: 0
Virtual address: 17298 Physical address: 22674 Value: 16
Virtual address: 56049 Physical address: 23025 Value: 0
Virtual address: 24768 Physical address: 23232 Value: 0
Virtual address: 54014 Physical address: 23550 Value: 52
Virtual address: 30875 Physical address: 14491 Value: 38
Virtual address: 26739 Physical address: 23667 Value: 28
Virtual address: 39456 Physical address: 23840 Value: 0
Virtual address: 18074 Physical address: 24218 Value: 17
Virtual address: 53349 Physical address: 2661 Value: 0
Virtual address: 33455 Physical address: 24495 Value: -85
Virtual address: 15853 Physical address: 24813 Value: 0
Virtual address: 6787 Physical address: 12419 Value: -96
Virtual address: 39639 Physical address: 24023 Value: -75
Virtual address: 37050 Physical address: 25018 Value: 36
Virtual address: 8789 Physical address: 25173 Value: 0
Virtual address: 17573 Physical address: 4261 Value: 0
Virtual address: 45940 Physical address: 15220 Value: 0
Virtual address: 53677 Physical address: 25517 Value: 0
Virtual address: 48740 Physical address: 25700 Value: 0
Virtual address: 41547 Physical address: 15947 Value: -110
Virtual address: 25791 Physical address: 26047 Value: 47
Virtual address: 58638 Physical address: 26126 Value: 57
Virtual address: 59213 Physical address: 26445 Value: 0
Virtual address: 35582 Physical address: 26878 Value: 34
Virtual address: 47319 Physical address: 27095 Value: 53
Virtual address: 51490 Physical address: 27170 Value: 50
Virtual address: 51246 Physical address: 27438 Value: 50
Virtual address: 7813 Physical address: 5765 Value: 0
Virtual address: 31637 Physical address: 27797 Value: 0
Virtual address: 40146 Physical address: 28114 Value: 39
Virtual address: 55051 Physical address: 9739 Value: -62
Virtual address: 31511 Physical address: 27671 Value: -59
Virtual address: 6871 Physical address: 12503 Value: -75
Virtual address: 38475 Physical address: 28235 Value: -110
Virtual address: 48768 Physical address: 25728 Value: 0
Virtual address: 1151 Physical address: 28543 Value: 31
Virtual address: 48552 Physical address: 28840 Value: 0
Virtual address: 19857 Physical address: 657 Value: 0
Virtual address: 13291 Physical address: 10219 Value: -6
Virtual address: 19171 Physical address: 29155 Value: -72
Virtual address: 63120 Physical address: 29328 Value: 0
Virtual address: 37021 Physical address: 24989 Value: 0
Virtual address: 59139 Physical address: 26371 Value: -64
Virtual address: 54146 Physical address: 29570 Value: 52
Virtual address: 32135 Physical address: 29831 Value: 97
Virtual address: 12845 Physical address: 29997 Value: 0
Virtual address: 55657 Physical address: 20329 Value: 0
Virtual address: 2036 Physical address: 18676 Value: 0
Virtual address: 35030 Physical address: 30422 Value: 34
Virtual address: 20139 Physical address: 30635 Value: -86
Virtual address: 40865 Physical address: 30881 Value: 0
Virtual address: 24264 Physical address: 31176 Value: 0
Virtual address: 38656 Physical address: 31232 Value: 0
Virtual address: 58202 Physical address: 31578 Value: 56
Virtual address: 726 Physical address: 21462 Value: 0
Virtual address: 45856 Physical address: 15136 Value: 0
Virtual address: 14356 Physical address: 10260 Value: 0
Virtual address: 43220 Physical address: 31956 Value: 0
Virtual address: 5740 Physical address: 32108 Value: 0
Virtual address: 54735 Physical address: 32463 Value: 115
Virtual address: 31647 Physical address: 27807 Value: -25
Virtual address: 2537 Physical address: 32745 Value: 0
Virtual address: 3605 Physical address: 1301 Value: 0
Virtual address: 25660 Physical address: 25916 Value: 0
Virtual address: 5777 Physical address: 32145 Value: 0
Virtual address: 34708 Physical address: 32916 Value: 0
Virtual address: 45763 Physical address: 11971 Value: -80
Virtual address: 11509 Physical address: 33269 Value: 0
Virtual address: 52424 Physical address: 33480 Value: 0
Virtual address: 63520 Physical address: 33568 Value: 0
Virtual address: 49272 Physical address: 33912 Value: 0
Virtual address: 54153 Physical address: 29577 Value: 0
Virtual address: 54439 Physical address: 34215 Value: 41
Virtual address: 27949 Physical address: 16685 Value: 0
Virtual address: 19423 Physical address: 34527 Value: -9
Virtual address: 51785 Physical address: 34633 Value: 0
Virtual address: 15489 Physical address: 34945 Value: 0
Virtual address: 47636 Physical address: 35092 Value: 0
Virtual address: 38338 Physical address: 35522 Value: 37
Virtual address: 47059 Physical address: 16595 Value: -12
Virtual address: 19917 Physical address: 717 Value: 0
Virtual address: 65441 Physical address: 3489 Value: 0
Virtual address: 3844 Physical address: 16900 Value: 0
Virtual address: 40724 Physical address: 30740 Value: 0
Virtual address: 24383 Physical address: 35647 Value: -49
Virtual address: 27178 Physical address: 12586 Value: 26
Virtual address: 54217 Physical address: 29641 Value: 0
Virtual address: 20514 Physical address: 35874 Value: 20
Virtual address: 12949 Physical address: 30101 Value: 0
Virtual address: 34521 Physical address: 11737 Value: 0
Virtual address: 46920 Physical address: 16456 Value: 0
Virtual address: 472 Physical address: 20696 Value: 0
Virtual address: 27647 Physical address: 36351 Value: -1
Virtual address: 38204 Physical address: 35388 Value: 0
Virtual address: 22993 Physical address: 12241 Value: 0
Virtual address: 30366 Physical address: 36510 Value: 29
Virtual address: 36134 Physical address: 21542 Value: 35
Virtual address: 50880 Physical address: 36800 Value: 0
Virtual address: 46120 Physical address: 36904 Value: 0
Virtual address: 33969 Physical address: 8881 Value: 0
Virtual address: 9414 Physical address: 1222 Value: 9
Virtual address: 62325 Physical address: 37237 Value: 0
Virtual address: 19229 Physical address: 34333 Value: 0
Virtual address: 14920 Physical address: 37448 Value: 0
Virtual address: 51196 Physical address: 6908 Value: 0
Virtual address: 61820 Physical address: 37756 Value: 0
Virtual address: 44765 Physical address: 38109 Value: 0
Virtual address: 47671 Physical address: 35127 Value: -115
Virtual address: 41094 Physical address: 38278 Value: 40
Virtual address: 53559 Physical address: 25399 Value: 77
Virtual address: 23793 Physical address: 38641 Value: 0
Virtual address: 16221 Physical address: 38749 Value: 0
Virtual address: 63979 Physical address: 39147 Value: 122
Virtual address: 21677 Physical address: 2221 Value: 0
Virtual address: 60744 Physical address: 18248 Value: 0
Virtual address: 1858 Physical address: 18498 Value: 1
Virtual address: 47532 Physical address: 39340 Value: 0
Virtual address: 4944 Physical address: 39504 Value: 0
Virtual address: 12918 Physical address: 30070 Value: 12
Virtual address: 35987 Physical address: 39827 Value: 36
Virtual address: 32478 Physical address: 40158 Value: 31
Virtual address: 61740 Physical address: 37676 Value: 0
Virtual address: 12272 Physical address: 40432 Value: 0
Virtual address: 46100 Physical address: 36884 Value: 0
Virtual address: 35055 Physical address: 30447 Value: 59
Virtual address: 62665 Physical address: 13513 Value: 0
Virtual address: 48604 Physical address: 28892 Value: 0
Virtual address: 9552 Physical address: 40528 Value: 0
Virtual address: 24770 Physical address: 23234 Value: 24
Virtual address: 19483 Physical address: 40731 Value: 6
Virtual address: 7102 Physical address: 41150 Value: 6
Virtual address: 13961 Physical address: 15497 Value: 0
Virtual address: 44326 Physical address: 806 Value: 43
Virtual address: 58725 Physical address: 26213 Value: 0
Virtual address: 19951 Physical address: 751 Value: 123
Virtual address: 50642 Physical address: 5586 Value: 49
Virtual address: 8794 Physical address: 25178 Value: 8
Virtual address: 21858 Physical address: 41314 Value: 21
Virtual address: 37531 Physical address: 41627 Value: -90
Virtual address: 58484 Physical address: 41844 Value: 0
Virtual address: 6437 Physical address: 42021 Value: 0
Virtual address: 44053 Physical address: 42261 Value: 0
Virtual address: 25360 Physical address: 42512 Value: 0
Virtual address: 8444 Physical address: 43004 Value: 0
Virtual address: 46509 Physical address: 43181 Value: 0
Virtual address: 45701 Physical address: 11909 Value: 0
Virtual address: 2140 Physical address: 43356 Value: 0
Virtual address: 45238 Physical address: 43702 Value: 44
Virtual address: 34737 Physical address: 32945 Value: 0
Virtual address: 35076 Physical address: 43780 Value: 0
Virtual address: 53149 Physical address: 44189 Value: 0
Virtual address: 12025 Physical address: 20985 Value: 0
Virtual address: 20907 Physical address: 11179 Value: 106
Virtual address: 37436 Physical address: 41532 Value: 0
Virtual address: 23285 Physical address: 17909 Value: 0
Virtual address: 14268 Physical address: 44476 Value: 0
Virtual address: 18430 Physical address: 44798 Value: 17
Virtual address: 52819 Physical address: 44883 Value: -108
Virtual address: 9365 Physical address: 1173 Value: 0
Virtual address: 33346 Physical address: 24386 Value: 32
Virtual address: 62705 Physical address: 13553 Value: 0
Virtual address: 39351 Physical address: 45239 Value: 109
Virtual address: 10389 Physical address: 11413 Value: 0
Virtual address: 59313 Physical address: 26545 Value: 0
Virtual address: 13892 Physical address: 15428 Value: 0
Virtual address: 35162 Physical address: 43866 Value: 34
Virtual address: 37132 Physical address: 45324 Value: 0
Virtual address: 2473 Physical address: 32681 Value: 0
Virtual address: 44561 Physical address: 37905 Value: 0
Virtual address: 51165 Physical address: 6877 Value: 0
Virtual address: 15838 Physical address: 24798 Value: 15
Virtual address: 44148 Physical address: 42356 Value: 0
Virtual address: 44315 Physical address: 795 Value: 70
Virtual address: 40089 Physical address: 28057 Value: 0
Virtual address: 39125 Physical address: 13013 Value: 0
Virtual address: 58509 Physical address: 41869 Value: 0
Virtual address: 56116 Physical address: 45620 Value: 0
Virtual address: 20035 Physical address: 30531 Value: -112
Virtual address: 24749 Physical address: 23213 Value: 0
Virtual address: 50348 Physical address: 4012 Value: 0
Virtual address: 53143 Physical address: 44183 Value: -27
Virtual address: 38211 Physical address: 35395 Value: 80
Virtual address: 9093 Physical address: 14981 Value: 0
Virtual address: 24106 Physical address: 31018 Value: 23
Virtual address: 31134 Physical address: 45982 Value: 30
Virtual address: 32123 Physical address: 29819 Value: 94
Virtual address: 29968 Physical address: 46096 Value: 0
Virtual address: 13339 Physical address: 7451 Value: 6
Virtual address: 4362 Physical address: 19978 Value: 4
Virtual address: 184 Physical address: 21944 Value: 0
Virtual address: 59291 Physical address: 26523 Value: -26
Virtual address: 37713 Physical address: 46417 Value: 0
Virtual address: 49461 Physical address: 2869 Value: 0
Virtual address: 3450 Physical address: 46714 Value: 3
Virtual address: 12594 Physical address: 46898 Value: 12
Virtual address: 62308 Physical address: 37220 Value: 0
Virtual address: 35283 Physical address: 43987 Value: 116
Virtual address: 57957 Physical address: 17509 Value: 0
Virtual address: 37593 Physical address: 41689 Value: 0
Virtual address: 28456 Physical address: 47144 Value: 0
Virtual address: 4782 Physical address: 47534 Value: 4
Virtual address: 2101 Physical address: 43317 Value: 0
Virtual address: 42719 Physical address: 47839 Value: -73
Virtual address: 46881 Physical address: 16417 Value: 0
Virtual address: 59368 Physical address: 26600 Value: 0
Virtual address: 11245 Physical address: 7149 Value: 0
Virtual address: 57061 Physical address: 48101 Value: 0
Virtual address: 37252 Physical address: 45444 Value: 0
Virtual address: 23343 Physical address: 48175 Value: -53
Virtual address: 56032 Physical address: 23008 Value: 0
Virtual address: 51474 Physical address: 27154 Value: 50
Virtual address: 8888 Physical address: 25272 Value: 0
Virtual address: 22768 Physical address: 1776 Value: 0
Virtual address: 35505 Physical address: 26801 Value: 0
Virtual address: 20857 Physical address: 11129 Value: 0
Virtual address: 16746 Physical address: 4458 Value: 16
Virtual address: 57259 Physical address: 48555 Value: -22
Virtual address: 63278 Physical address: 1838 Value: 61
Virtual address: 40674 Physical address: 48866 Value: 39
Virtual address: 4010 Physical address: 17066 Value: 3
Virtual address: 27403 Physical address: 36107 Value: -62
Virtual address: 45576 Physical address: 11784 Value: 0
Virtual address: 17005 Physical address: 49005 Value: 0
Virtual address: 52621 Physical address: 49293 Value: 0
Virtual address: 38644 Physical address: 28404 Value: 0
Virtual address: 32869 Physical address: 22373 Value: 0
Virtual address: 33572 Physical address: 13092 Value: 0
Virtual address: 39304 Physical address: 45192 Value: 0
Virtual address: 24113 Physical address: 31025 Value: 0
Virtual address: 44856 Physical address: 49464 Value: 0
Virtual address: 13249 Physical address: 10177 Value: 0
Virtual address: 33733 Physical address: 13253 Value: 0
Virtual address: 12786 Physical address: 47090 Value: 12
Virtual address: 24234 Physical address: 31146 Value: 23
Virtual address: 26670 Physical address: 23598 Value: 26
Virtual address: 64792 Physical address: 49688 Value: 0
Virtual address: 11859 Physical address: 20819 Value: -108
Virtual address: 58281 Physical address: 31657 Value: 0
Virtual address: 26897 Physical address: 49937 Value: 0
Virtual address: 5900 Physical address: 50188 Value: 0
Virtual address: 6146 Physical address: 50434 Value: 6
Virtual address: 22445 Physical address: 50861 Value: 0
Virtual address: 22752 Physical address: 1760 Value: 0
Virtual address: 59295 Physical address: 26527 Value: -25
Virtual address: 43031 Physical address: 31767 Value: 5
Virtual address: 50880 Physical address: 36800 Value: 0
Virtual address: 15100 Physical address: 37628 Value: 0
Virtual address: 20202 Physical address: 30698 Value: 19
Virtual address: 40061 Physical address: 28029 Value: 0
Virtual address: 61616 Physical address: 51120 Value: 0
Virtual address: 30399 Physical address: 36543 Value: -81
Virtual address: 14935 Physical address: 37463 Value: -107
Virtual address: 26455 Physical address: 51287 Value: -43
Virtual address: 56896 Physical address: 47936 Value: 0
Virtual address: 33636 Physical address: 13156 Value: 0
Virtual address: 47147 Physical address: 26923 Value: 10
Virtual address: 38552 Physical address: 28312 Value: 0
Virtual address: 57676 Physical address: 51532 Value: 0
Virtual address: 2052 Physical address: 43268 Value: 0
Virtual address: 25769 Physical address: 26025 Value: 0
Virtual address: 2418 Physical address: 32626 Value: 2
Virtual address: 53593 Physical address: 25433 Value: 0
Virtual address: 13658 Physical address: 51802 Value: 13
Virtual address: 60795 Physical address: 18299 Value: 94
Virtual address: 21013 Physical address: 51989 Value: 0
Virtual address: 57971 Physical address: 17523 Value: -100
Virtual address: 19432 Physical address: 34536 Value: 0
Virtual address: 24339 Physical address: 35603 Value: -60
Virtual address: 205 Physical address: 21965 Value: 0
Virtual address: 44654 Physical address: 37998 Value: 43
Virtual address: 44801 Physical address: 49409 Value: 0
Virtual address: 16501 Physical address: 52341 Value: 0
Virtual address: 64965 Physical address: 49861 Value: 0
Virtual address: 24141 Physical address: 31053 Value: 0
Virtual address: 53878 Physical address: 23414 Value: 52
Virtual address: 15409 Physical address: 34865 Value: 0
Virtual address: 14377 Physical address: 10281 Value: 0
Virtual address: 52234 Physical address: 33290 Value: 51
Virtual address: 38561 Physical address: 28321 Value: 0
Virtual address: 24164 Physical address: 31076 Value: 0
Virtual address: 24918 Physical address: 52566 Value: 24
Virtual address: 10119 Physical address: 52871 Value: -31
Virtual address: 14400 Physical address: 10304 Value: 0
Virtual address: 27557 Physical address: 36261 Value: 0
Virtual address: 17704 Physical address: 6184 Value: 0
Virtual address: 36327 Physical address: 21735 Value: 121
Virtual address: 13037 Physical address: 30189 Value: 0
Virtual address: 38738 Physical address: 31314 Value: 37
Virtual address: 9743 Physical address: 53007 Value: -125
Virtual address: 53050 Physical address: 44090 Value: 51
Virtual address: 46871 Physical address: 16407 Value: -59
Virtual address: 7207 Physical address: 53287 Value: 9
Virtual address: 50939 Physical address: 36859 Value: -66
Virtual address: 11736 Physical address: 53720 Value: 0
Virtual address: 16581 Physical address: 52421 Value: 0
Virtual address: 19156 Physical address: 29140 Value: 0
Virtual address: 16622 Physical address: 52462 Value: 16
Virtual address: 64078 Physical address: 7758 Value: 62
Virtual address: 31990 Physical address: 54006 Value: 31
Virtual address: 48570 Physical address: 28858 Value: 47
Virtual address: 33282 Physical address: 24322 Value: 32
Virtual address: 18993 Physical address: 28977 Value: 0
Virtual address: 24493 Physical address: 35757 Value: 0
Virtual address: 23873 Physical address: 54081 Value: 0
Virtual address: 23682 Physical address: 38530 Value: 23
Virtual address: 33273 Physical address: 15865 Value: 0
Virtual address: 44298 Physical address: 778 Value: 43
Virtual address: 23857 Physical address: 54065 Value: 0
Virtual address: 7966 Physical address: 54302 Value: 7
Virtual address: 35322 Physical address: 44026 Value: 34
Virtual address: 33850 Physical address: 8762 Value: 33
Virtual address: 5380 Physical address: 54532 Value: 0
Virtual address: 33088 Physical address: 15680 Value: 0
Virtual address: 46587 Physical address: 43259 Value: 126
Virtual address: 59413 Physical address: 54805 Value: 0
Virtual address: 64318 Physical address: 55102 Value: 62
Virtual address: 52462 Physical address: 33518 Value: 51
Virtual address: 36879 Physical address: 24847 Value: 3
Virtual address: 10311 Physical address: 11335 Value: 17
Virtual address: 25840 Physical address: 26096 Value: 0
Virtual address: 3808 Physical address: 1504 Value: 0
Virtual address: 26529 Physical address: 51361 Value: 0
Virtual address: 46611 Physical address: 55315 Value: -124
Virtual address: 48451 Physical address: 28739 Value: 80
Virtual address: 52901 Physical address: 44965 Value: 0
Virtual address: 2660 Physical address: 8036 Value: 0
Virtual address: 53150 Physical address: 44190 Value: 51
Virtual address: 27795 Physical address: 10899 Value: 36
Virtual address: 56085 Physical address: 45589 Value: 0
Virtual address: 27642 Physical address: 36346 Value: 26
Virtual address: 65463 Physical address: 3511 Value: -19
Virtual address: 29674 Physical address: 18922 Value: 28
Virtual address: 35092 Physical address: 43796 Value: 0
Virtual address: 61165 Physical address: 493 Value: 0
Virtual address: 60381 Physical address: 55773 Value: 0
Virtual address: 24381 Physical address: 35645 Value: 0
Virtual address: 15497 Physical address: 34953 Value: 0
Virtual address: 28240 Physical address: 55888 Value: 0
Virtual address: 63968 Physical address: 39136 Value: 0
Virtual address: 51085 Physical address: 6797 Value: 0
Virtual address: 46685 Physical address: 55389 Value: 0
Virtual address: 28411 Physical address: 56059 Value: -66
Virtual address: 8859 Physical address: 25243 Value: -90
Virtual address: 48885 Physical address: 25845 Value: 0
Virtual address: 54527 Physical address: 34303 Value: 63
Virtual address: 61740 Physical address: 37676 Value: 0
Virtual address: 15613 Physical address: 35069 Value: 0
Virtual address: 19570 Physical address: 40818 Value: 19
Virtual address: 25976 Physical address: 56184 Value: 0
Virtual address: 11732 Physical address: 53716 Value: 0
Virtual address: 21739 Physical address: 2283 Value: 58
Virtual address: 35334 Physical address: 26630 Value: 34
Virtual address: 19955 Physical address: 755 Value: 124
Virtual address: 14844 Physical address: 56572 Value: 0
Virtual address: 18082 Physical address: 24226 Value: 17
Virtual address: 35060 Physical address: 30452 Value: 0
Virtual address: 31986 Physical address: 54002 Value: 31
Virtual address: 7320 Physical address: 53400 Value: 0
Virtual address: 15501 Physical address: 34957 Value: 0
Virtual address: 64304 Physical address: 55088 Value: 0
Virtual address: 58953 Physical address: 56649 Value: 0
Virtual address: 44322 Physical address: 802 Value: 43
Virtual address: 31275 Physical address: 9003 Value: -118
Virtual address: 65187 Physical address: 56995 Value: -88
Virtual address: 30582 Physical address: 21110 Value: 29
Virtual address: 50947 Physical address: 6659 Value: -64
Virtual address: 50486 Physical address: 5430 Value: 49
Virtual address: 13876 Physical address: 15412 Value: 0
Virtual address: 50423 Physical address: 4087 Value: 61
Virtual address: 37916 Physical address: 57116 Value: 0
Virtual address: 65292 Physical address: 3340 Value: 0
Virtual address: 63584 Physical address: 33632 Value: 0
Virtual address: 365 Physical address: 20589 Value: 0
Virtual address: 17461 Physical address: 4149 Value: 0
Virtual address: 45267 Physical address: 43731 Value: 52
Virtual address: 18051 Physical address: 24195 Value: -96
Virtual address: 42832 Physical address: 80 Value: 0
Virtual address: 51613 Physical address: 27293 Value: 0
Virtual address: 64957 Physical address: 49853 Value: 0
Virtual address: 60294 Physical address: 55686 Value: 58
Virtual address: 47111 Physical address: 26887 Value: 1
Virtual address: 25648 Physical address: 25904 Value: 0
Virtual address: 25729 Physical address: 25985 Value: 0
Virtual address: 38780 Physical address: 31356 Value: 0
Virtual address: 32459 Physical address: 40139 Value: -78
Virtual address: 37672 Physical address: 46376 Value: 0
Virtual address: 16579 Physical address: 52419 Value: 48
Virtual address: 23146 Physical address: 17770 Value: 22
Virtual address: 63537 Physical address: 33585 Value: 0
Virtual address: 26950 Physical address: 49990 Value: 26
Virtual address: 12588 Physical address: 46892 Value: 0
Number of Translated Addresses = 500
Page Faults = 224
Page Fault Rate = 0.448
TLB Hits = 30
TLB Hit Rate = 0.060
